### PR TITLE
Create category card

### PR DESCRIPTION
**Pull request by ch55secake**
Added accordion component for lists and shit heres what it looks like 
image

### CODE REVIEW

**Comment by h0bnobs** [comment]
I ain't readin... allat 💯

**Review by h0bnobs — COMMENTED** [review]
https://www.youtube.com/shorts/MtaiRXCvMcU

File: frontend/src/components/collapsible/collapsible.module.css [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+.content {
+    color: white;
+    max-height: 0;
+    overflow: hidden;
+    transition: all 0.5s cubic-bezier(0, 1, 0, 1);
+}
+
+.contentShow {
+    height: auto;
+    max-height: 9999px;
```
Comment (LP2851) [inline frontend/src/components/collapsible/collapsible.module.css:34]: I wouldn't think this is needed to work

File: frontend/src/containers/home/category-card/index.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
+import CardComponent from "../../../components/card/Card";
+import {memo} from "react";
+import styles from "./index.module.css"
```
Comment (LP2851) [inline frontend/src/containers/home/category-card/index.js]: Styles should be the last thing in the import list- just being picky

**Review by LP2851 — CHANGES_REQUESTED** [review]
Just some small changes needed- but looks really cool!

File: frontend/src/components/collapsible/collapsible.module.css (hunk 1 above)
Comment (ch55secake) [reply frontend/src/components/collapsible/collapsible.module.css:34]: we dont but without it we dont get the cool transition effect

**Review by LP2851 — APPROVED** [review]

**Pull request merged**